src/crimson/os/seastore/seastore.cc

```diff
@@ -711,7 +711,8 @@ SeaStore::Shard::list_objects(CollectionRef ch,
 		DEBUGT("pstart {}, pend {}, limit {}", t, pstart, pend, limit);
 		return onode_manager->list_onodes(
 		  t, pstart, pend, limit
-		).si_then([&limit, &ret, pend](auto &&_ret) mutable {
+		).si_then([&limit, &ret, pend, &t, last=ranges.empty()]
+			  (auto &&_ret) mutable {
```

Comment:
last=ranges.empty() is handled on line 701

Reply:
yeah, will correct it.